src/hotspot/share/opto/vectornode.cpp

```diff
@@ -1042,10 +1042,69 @@ Node* VectorNode::try_to_gen_masked_vector(PhaseGVN* gvn, Node* node, const Type
   }
 }
 
+bool VectorNode::should_swap_inputs() {
```

Comment:
```suggestion
// If we have a AddVB(v1, v2) and AddVB(v2, v1), we want to swap the edges of one of them
// so that they become identical, and can common in global value numbering.
bool VectorNode::should_swap_inputs_to_help_global_value_numbering() {
```